update-tags.mjs

```diff
@@ -0,0 +1,65 @@
+import { readFile, writeFile } from 'fs/promises'
+import { execSync } from 'child_process'
```

Comment:
You're already using promises. Why not execute external processes asynchronously too?